templates/chat/src/app/page.tsx

```diff
@@ -0,0 +1,14 @@
+'use client'
+import { TldrawUiTooltipProvider } from 'tldraw'
+import { Chat } from '../components/Chat'
+
+export default function Home() {
+	return (
+		<main className="tl-container tl-theme__light">
```

Comment:
took me a sec to figure this out, annoying thing about `tl-container` is that it sets `user-select: none` which prevents copy/pasting text from the chat. let's override that pls!